blueprint/index.md

```diff
@@ -54,10 +62,11 @@ The Chat Assistant integration has the following stages:
    
    Be sure to also add the URL parameter pcConversationId at the end of the URL. This ensures that the solution will pass the active conversation ID to the solution.
    ```
-   /?conversationid={pcConversationId}
+   /?conversationid={{pcConversationId}
```

Comment:
Using the double curly braces causes an error within the Genesys site. Tim Smith has pushed a workaround to still reflect the double curly braces.